src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,54 @@
+package lotto.domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Lotto {
+
+  private Set<LottoNumber> ticket = new HashSet<>();
```

Comment:
- 적절한 자료구조 사용 👍 
- ticket은 외부에서 항상 주입받아서 굳이 선언문에서 인스턴스를 생성할 필요 없겠네요.
    ```suggestion
      private Set<LottoNumber> ticket;
```